src/controllers/sessionController.ts

```diff
@@ -67,11 +68,22 @@ export default class SessionController {
         TE.toUnion
       )();
 
+      // Read the assertionRef related to the User for Lollipop.
+      const maybeAssertionRef = await this.sessionStorage.getLollipopAssertionRefForUser(
```

Comment:
do you think `errorOrMaybeAssertionRef` can be a more appropriate variable name here?

Reply:
Fixed in [d15504c](https://github.com/pagopa/io-backend/pull/993/commits/d15504cccb069578e7702f32563b1837194be103)